### PR TITLE
add some documentation and fix the load balancer naming

### PR DESCRIPTION
**Pull request by tariq1890**
Some minor fixes to the documentation and renaming of `loadbalancers` to `loadBalancers`

### CODE REVIEW

File: docs/controllers/services/annotations.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Specifies which algorithm the Load Balancer should use. Options are `round_robin
 
 Specifies which stick session type the loadbalancer should use. Options are `none` or `cookies`.
 
+### service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-cookie-name
+
+Specfies what cookie name to use for the DO loadbalancer sticky session. This annotation is required if `service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-type` is set to `cookies`.
```
Comment (timoreimann) [inline docs/controllers/services/annotations.md]:
```suggestion
Specifies what cookie name to use for the DO load balancer sticky session. This annotation is required if `service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-type` is set to `cookies`.
```

File: docs/controllers/services/annotations.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ Specifies which algorithm the Load Balancer should use. Options are `round_robin
...
+
 ### service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-cookie-ttl
 
 Specifies the TTL of cookies used for loadbalancer sticky sessions. This annotation is required if `service.beta.kubernetes.io/do-loadbalancer-sticky-sessions-type` is set.
```
Comment (timoreimann) [inline docs/controllers/services/annotations.md]: Would you mind appending

```
to `cookies`
```

to the last sentence in order to be consistent with your added paragraph above?

Comment (tariq1890) [reply docs/controllers/services/annotations.md]: @timoreimann I am not sure I understand this comment.

Comment (tariq1890) [reply docs/controllers/services/annotations.md]: Oh nvm I got it

**Comment by tariq1890** [comment]
@timoreimann Review comments addressed.

**Pull request merged**